src/api.cc

```diff
@@ -172,6 +172,25 @@ TufTarget AkliteClient::GetCurrent() const {
   return TufTarget(current.filename(), current.sha256Hash(), ver, current.custom_data());
 }
 
+DeviceResult AkliteClient::GetDevice() const {
+  DeviceResult res;
+  res.status = DeviceResult::Status::Failed;
```

Comment:
I think these two line can be replaced with `DeviceResult res{DeviceResult::Status::Failed};`

Reply:
thanks. fixed in force-push